tests/examples/opinions/united_states/nmariana_example.compare.json

```diff
@@ -7,8 +7,14 @@
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
     "docket_numbers": "",
+    "judges": [
```

Comment:
how do we get judges here and judge elsewhere

Reply:
There is a single return interface, defined by AbstractSite. We get should be getting "judges" as a key everywhere, because Courtlistener never uses a "judge" key

Apart from that, it shouldn't return a list, but I wasn't checking values, only checking key names. The list type comes from way back
https://github.com/freelawproject/juriscraper/commit/2ec36d8d7ef02ef171e348ee2369a20209e2d4d4

Reply:
okay, so this is a mistake (presumably I made) here but it shouldn't affect this pr